packages/gestalt/src/SideNavigationNestedItem.tsx

```diff
@@ -14,10 +14,18 @@ type Props = {
     number: string;
     accessibilityLabel: string;
   };
+  /**
+   * When supplied, will gray out the item and make it non-clickable
+   */
+  disabled?: boolean;
   /**
    * Directs users to the url when item is selected.
    */
   href: string;
+  /**
+   * Optional subtext for the item label
```

Comment:
see: Optional "subtext". i think subtext makes more sense to describe this

Reply:
follow this descripion patterns: https://deploy-preview-3905--gestalt.netlify.app/web/button


Indicates if NestedItem is disabled. Disabled Items are inactive and cannot be interacted with. See the state variant for details on proper usage.
--


<br class="Apple-interchange-newline">Indicates if Button is disabled. Disabled Buttons are inactive and cannot be interacted with. See the [state variant](https://deploy-preview-3905--gestalt.netlify.app/web/button#State) for details on proper usage.